ticket_group_details.view.lkml

```diff
@@ -0,0 +1,60 @@
+view: ticket_group_details {
+  derived_table: {
+    sql: SELECT
+        DISTINCT audits.ticket_id AS ticket_id,
+        groups.name AS first_group,
+        events.value AS first_group_id
+      FROM
+        zendesk_stitch.ticket_audits__events AS events
+      LEFT JOIN
+        zendesk_stitch.ticket_audits AS audits
+      ON
+        audits.id = events._sdc_source_key_id
+      LEFT JOIN
+        zendesk_stitch.groups AS groups
+      ON
+        groups.id::text = events.value
+      WHERE
+        events.field_name = 'group_id'
+        AND events.type = 'Create'
+       ;;
+    indexes: ["ticket_id"]
+    sql_trigger_value: SELECT COUNT(*) FROM zendesk_stitch.ticket_audits__events ;;
+  }
+
+  dimension: ticket_id {
+    primary_key: yes
+    type: number
+    hidden: yes
+    sql: ${TABLE}.ticket_id ;;
+  }
+
+  dimension: first_group {
+    description: "The name of the first group the ticket was assigned to"
+    group_label: "First Touch"
+    type: string
+    sql: ${TABLE}.first_group ;;
+  }
+
+  dimension: last_group {
```

Comment:
Isn't `last_group` redundant as that should be the current state of the ticket, no?

Reply:
Correct in that is should always be the same as the current state of the ticket.  I'd like to keep it though so that there isn't confusion about why there is the first group available but not the last group.  See view visual here --
<img width="330" alt="screen shot 2018-11-21 at 3 19 13 pm" src="https://user-images.githubusercontent.com/38991765/48873065-e954d000-eda0-11e8-9e0d-20490d61946b.png">